src/state/nftMarket/queries.ts

```diff
@@ -7,4 +7,22 @@ export const getBaseNftsFields = () => `
   tradeVolumeBNB
   totalTrades
   isTradable
+  collection {
+    id
```

Comment:
I don't think we should include sub queries in the base fields.

Reply:
Ah I hear ya - this one in particular is pretty key for every NFT query though. We can refactor later?